ext/phar/util.c

```diff
@@ -256,17 +256,23 @@ zend_string *phar_find_in_include_path(zend_string *filename, phar_archive_data
 		return NULL;
 	}
 
-	fname = (char*)zend_get_executed_filename();
-	fname_len = strlen(fname);
+	zend_string *fname = zend_get_executed_filename_ex();
```

Comment:
Contrary to `zend_get_executed_filename()` `zend_get_executed_filename_ex()` may return `NULL`. I don't know if this code can be triggered outside of user context. The same goes for the calls above.